src/containers/my-cooperations/cooperation-notes/create-or-edit-note/CreateOrEditNote.tsx

```diff
@@ -55,18 +78,33 @@ const CreateOrEditNote = ({
     }
   })
 
+  const userPhoto = photo
+    ? `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`
```

Comment:
What do you think about using `createUrlPath` helper function instead? But it requires a small fix inside a helper so that we don't worry about adding a slash or not between parts of url. It was discussed earlier, but nobody take responsibility for this  huge improvement

Reply:
Refactored it with new URL()